ocs_ci/utility/kms.py

```diff
@@ -1039,7 +1041,176 @@ def create_vault_kube_auth_role(
             logger.info(f"Role {role_name} created successfully")
 
 
-kms_map = {"vault": Vault}
+class Hpcs(KMS):
```

Comment:
I think class name should be all caps because its in abbreviated from
```suggestion
class HPCS(KMS):
```